### PR TITLE
Commerical button icons

### PR DESCRIPTION
**Pull request by phamann**
This ports all of the commercial component buttons to use inline SVGs over our old `<i>` pattern. 
- refactor components to reference inline svg fragment
- refactor CSS to style new icons and change colours
- bump pasteup-buttons dep to correctly size icons within buttons
- Test on the `/commercial/test-page` page.

This is the first step to move facia away from the old icons system.

/cc @austinh7 @sndrs

### CODE REVIEW

File: commercial/app/views/books/book.scala.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h4 class="lineitem__title" itemprop="name">@book.title</h4>
             </p>
             </a>
             <a href="@clickMacro@book.buyUrl" class="lineitem__cta button button--primary button--small" data-link-name="merchandising-bookshop-v@{version}_@{date}-superHigh-visit-shop">
-                Buy now<i class="i i-arrow-white-right i-right"></i>
+                Buy now@fragments.inlineSvg("arrow-white-right", "icon", List("i-right"))
```
Comment (sndrs) [inline commercial/app/views/books/book.scala.html]: plan was to use `inline-` prefix for all these, until we've removed everything from css and then revert all inlined icons (i.e. by then just 'icons') to the `i-` prefix. think it would be useful if it's simple, but it's not worth breaking backs over it if it's not straightforward.

Comment (sndrs) [inline commercial/app/views/books/book.scala.html]: should be just `arrow-right`, with colour applied by css. 

*ideally*, just `arrow`, with rotation also handled by css...?

Comment (phamann) [inline commercial/app/views/books/book.scala.html]: As discussed, I tried to avoid touching the commercial CSS as much as possible hence adding the `i-right` class to inherit the styles already applied to it for positioning. 

But do agree with the naming and colour convention so will re-name to `arrow-right` now. `arrow` and rotation should be a refactor in another PR.

**Comment by phamann** [comment]
@sndrs have now updated name and icon

**Comment by sndrs** [comment]
:+1:  lush
